partner_mobile_check/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Ashish (Ooops) <https://ooops404.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Mobile Duplicate Checker",
```

Comment:
"name": "Partner Mobile Format and Duplicate Checker",

Reply:
done